### PR TITLE
Timeslider so far and events

### PR DESCRIPTION
**Pull request by mwernimont**
@jiwalker-usgs added the JS and CSS files I needed to the viz.yaml, hopefully getting it in before merge conflicts

### CODE REVIEW

File: layout/js/app.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+$(document).ready(function () {
```
Comment (jiwalker-usgs) [inline layout/js/app.js:1]: maybe call this file slider.js to keep it modular

File: layout/js/app.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+	var slider = document.getElementById('slider');
+	
+	var range_all_sliders = {
+		'min': [1950],
+		'7.6923076923%': [1955, 500],
+		'15.384615385%': [1960, 1000],
+		'23.076923077%': [1965, 1500],
+		'30.769230769%': [1970, 2000],
+		'38.461538462%': [1975, 2500],
+		'46.153846154%': [1980, 500],
```
Comment (jiwalker-usgs) [inline layout/js/app.js:12]: Did you have to pull out the calculator for these?

**Review by jiwalker-usgs — APPROVED** [review]
Looks good

File: layout/js/app.js (hunk 2 above)
Comment (mwernimont) [reply layout/js/app.js:12]: I did, even spacing is key

**Pull request merged**